### PR TITLE
About-me section

### PR DESCRIPTION
**Pull request by RalucaRbc**

### CODE REVIEW

**Review by MariaTeo — APPROVED** [review]
💯

File: src/_asset/font/font.scss [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+.icon__book {
+  &:before {
+    content: $icon__book; 
+  }
+}
+.icon__codetap {
+  &:before {
+    content: $icon__codetap; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/font.scss:48]: It would be nice if you would leave an empty line inbetween

File: src/_asset/font/font.scss [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+.icon__codewars {
+  &:before {
+    content: $icon__codewars; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/font.scss:53]: Here as well... and so on, for readability. 
I hope you will not get mad 😅

**Review by MariaTeo — APPROVED** [review]

**Review by Constantin-Matei — APPROVED** [review]
Nice work

**Review by psyhedeliq — APPROVED** [review]
Nice work! 🚀

**Pull request merged**